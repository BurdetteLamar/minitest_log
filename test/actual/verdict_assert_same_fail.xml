<log>
  <summary_ verdicts='3' failures='2' errors='0'/>
  <section_ name='Test' method='verdict_assert_same?' type='fail'>
    <section_ name='Args=[&quot;foo&quot;, &quot;foo&quot;, &quot;Message 0&quot;]'>
      <verdict_ method='verdict_assert_same?' outcome='failed' id='0' message='Message 0'>
        <expected_ class='String' value='&quot;foo&quot;'/>
        <actual_ class='String' value='&quot;foo&quot;'/>
        <exception_ class='Minitest::Assertion' message='Expected # encoding: UTF-8'>
          <backtrace_>
          </backtrace_>
        </exception_>
      </verdict_>
    </section_>
    <section_ name='Args=[[], [], &quot;Message 1&quot;]'>
      <verdict_ method='verdict_assert_same?' outcome='failed' id='1' message='Message 1'>
        <expected_ class='Array' value='[]'/>
        <actual_ class='Array' value='[]'/>
        <exception_ class='Minitest::Assertion' message='Expected [] oid=(00000000) to be the same as [] oid=(00000000).'>
          <backtrace_>
          </backtrace_>
        </exception_>
      </verdict_>
    </section_>
  </section_>
  <verdict_ method='verdict_assert_equal?' outcome='passed' id='error_count'>
    <expected_ class='Integer' value='0'/>
    <actual_ class='Integer' value='0'/>
  </verdict_>
</log>
