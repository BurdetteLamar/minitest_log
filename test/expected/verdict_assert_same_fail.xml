<log>
  <summary verdicts='3' failures='2' errors='0'/>
  <section name='Test' method='verdict_assert_same?' type='fail'>
    <section name='Args=[&quot;foo&quot;, &quot;foo&quot;]'>
      <verdict method='verdict_assert_same?' outcome='failed' id='0' message='Message 0'>
        <expected class='String' value='&quot;foo&quot;'/>
        <actual class='String' value='&quot;foo&quot;'/>
        <exception class='Minitest::Assertion' message='Expected # encoding: UTF-8'>
          <backtrace/>
        </exception>
      </verdict>
    </section>
    <section name='Args=[[], []]'>
      <verdict method='verdict_assert_same?' outcome='failed' id='1' message='Message 1'>
        <expected class='Array' value='[]'/>
        <actual class='Array' value='[]'/>
        <exception class='Minitest::Assertion' message='Expected [] oid=(00000000) to be the same as [] oid=(00000000).'>
          <backtrace/>
        </exception>
      </verdict>
    </section>
  </section>
  <verdict method='verdict_assert_equal?' outcome='passed' id='__error_count__'>
    <expected class='Integer' value='0'/>
    <actual class='Integer' value='0'/>
  </verdict>
</log>
